cmd/git-init/main.go

```diff
@@ -87,16 +91,18 @@ func main() {
 		run(logger, "git", "init")
```

Comment:
Why you do that twice?

And perhaps check if path+"./git" exist already ?

Reply:
@chmouel it's not on the same "branch", so still needed